### ISSUE TITLE
Show "some green" for status full vacccined status

### ISSUE THREAD

**Issue by gizmo21** [corona-warn-app/cwa-wishlist#561]
Current Implementation

Right now a fully vaccinated after 2 weeks status shows a white shield on blue with check mark in main menu.
![1c9cf7f97-3c42-4334-adc6-fb1e1fa95ea9](https://user-images.githubusercontent.com/2779732/122834824-9a240280-d2ef-11eb-8823-6a16625bebe2.jpg)

It is nearly the same blue symbol as for adding the certs

![256353e47-0fb5-4635-b2b7-315f25351977](https://user-images.githubusercontent.com/2779732/122834906-ca6ba100-d2ef-11eb-829a-a2f16c02c319.jpg)

And even in the added certs details the second cert stays blue after the 2 weeks.

![36a9a6d51-52d8-479f-acca-441f37515da8](https://user-images.githubusercontent.com/2779732/122835031-f9821280-d2ef-11eb-9058-51eda31a6413.jpg)

All that even the whole is/was called the "digital green certificate", no enduser can on first sight see that oneself has the correct green status as blue suggests it could be or could be not.
And a line telling the time to full status is vanishing after changing to OK.

Suggested Enhancement
That textline should better change the text/color to a positive text and one or the other shield should turn into "OK green" instead of staying "undecided blue" even if it hurts UX designers eyes somekind.

Update: even the before undecided pcr test in blue, turns green when negative.

Expected Benefits
like in covpass check app the enduser also would clearly see that his own status is OK.

**Comment by jucktnich** [comment]
I think Green is reserved for negative tests

**Comment by Ein-Tim** [comment]
Yes the test certificate in version 2.4 will also be green: 

<img src="https://user-images.githubusercontent.com/67682506/122866549-703b0200-d328-11eb-9e5e-4c856356fc27.png" width="300">

**Comment by dsarkar** [comment]
@gizmo21 and everybody. I think it was someone of the community who already pointed out that it is not called Green Certificates anymore, but EU Digital Covid Certificates.

**Comment by Ein-Tim** [comment]
@dsarkar 

Yes, I also remember a comment from somebody reg. this, but I can't find it anymore... 😕

@gizmo21 I wonder if you know from a case where a real end user was confused by the _problem_ you showed above.

I also disagree that the box which is shown if there is no certificate added looks like the one that's shown if there is a certificate added. The box is black (in darkmode) or white (in lightmode) if there is no certificate registered. If there is one it's either gray (if you haven't received all shots yet or the last shot is not yet 14 days old) or completely blue.

**Comment by MikeMcC399** [comment]
> I think it was someone of the community who already pointed out that it is not called Green Certificates anymore, but EU Digital Covid Certificates.

I believe it was me 🙂 - see https://github.com/corona-warn-app/cwa-wishlist/issues/424 which links to https://www.europarl.europa.eu/news/en/press-room/20210422IPR02606/eu-covid-19-certificate-must-facilitate-free-movement-without-discrimination

The reference pages are:
- https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_en (English)
- https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_de (German)

**Comment by Jo-Achim** [comment]
For example, I would appreciate these changes for clarification:
 
> 
>![122866549-703b0200-d328-11eb-9e5e-4c856356fc27](https://user-images.githubusercontent.com/21124877/123241355-5c86cb80-d4e1-11eb-95e8-4a0824d7e57f.jpg)


However, if the display does not turn out like this, I would like to add the word "**complete**" (for a 'complete vaccination status') to the display.

In any case, the ad must be so clear that even people who have little to do with the CWA clearly understand the message of the ad. (See: CovPass, "Complete Vaccination Protection" below the QR code.)  

Best regards, Joachim.

**Comment by Ein-Tim** [comment]
FYI: Twitter user [reported](https://twitter.com/robertkoehh/status/1418112876485791748?s=21) that he was asked if he is fully vaccinated because the app showed the certificate in blue and not in green.

**Comment by Ein-Tim** [comment]
- This will be implemented (more or less) with version 2.27. Take a look at the screenshots in PR https://github.com/corona-warn-app/cwa-app-ios/pull/4758 for example.

**Comment by gizmo21** [comment]
Better late... thx
